src/main/java/com/smartbear/mqttsupport/teststeps/panels/PublishTestStepPanel.java

```diff
@@ -81,23 +81,23 @@ public PublishTestStepPanel(PublishTestStep modelItem) {
 
     private void buildUI() {
         JComponent mainPanel = buildMainPanel();
-        inspectorPanel = JInspectorPanelFactory.build(mainPanel);
+        inspectorPanel = JInspectorPanelFactory.buildRequestInspectorPanel(mainPanel, getModelItemSimpleName());
 
 
-        logInspector = new JComponentInspector<JComponent>(buildLogPanel(), String.format(LOG_TAB_TITLE, 0), "Log of the test step executions", true);
+        logInspector = new JComponentInspector<JComponent>(buildLogPanel(),
+                TestStepVariables.LOGGER_INSPECTOR_TITLE, TestStepVariables.LOGGER_INSPECTOR_DESCRIPTION, true);
         inspectorPanel.addInspector(logInspector);
 
-        inspectorPanel.setDefaultDividerLocation(0.6F);
-//        inspectorPanel.setCurrentInspector("Assertions");
-
-
         add(inspectorPanel.getComponent());
         setPreferredSize(new Dimension(500, 300));
 
     }
 
     private JComponent buildMainPanel() {
         JPanel root = new JPanel(new MigLayout("wrap", "0[grow,fill]0", "0[]0[grow,fill]0"));
+        root.setBorder(BorderFactory.createMatteBorder(
```

Comment:
Any reason for this?

Reply:
with these changes it looks better
![image](https://user-images.githubusercontent.com/7901884/114350220-964a6600-9b71-11eb-83e1-b30adf94c4f1.png)

Reply:
![image](https://user-images.githubusercontent.com/7901884/114350387-d0b40300-9b71-11eb-8c99-b17a8500be62.png)